### ISSUE TITLE
[Bug]: Can not run or install GhostBSD Acer Aspire 5 (A515-57-731E)

### ISSUE THREAD

**Issue by jespinal** [ghostbsd/ghostbsd-src#258]
### Version

GhostBSD-22.01.12.iso | GhostBSD-22.06.18.iso | GhostBSD-23.02.02.iso

### Explain the issue and what happens

I acquired a new laptop and attempted to install/run GhostBSD.

After the boot menu, it loads some components and when running `init.sh` it will always fail when trying to running the following command from the script:

`dd if=/cdrom/data/system.img status=progress bs=1M | zfs recv -F livecd`

...with the following error message:

```txt
cannot receive new filesystem stream: checksum mismatch
```

Initially, I thought that this might be due to some corruption in the ISO files, but the checksum of the downloaded files is fine. Then, I thought that this might be a version-specific issue, but the exact same thing happens with the following versions in 2 different USB sticks:

```txt
GhostBSD-22.01.12.iso
GhostBSD-22.06.18.iso
GhostBSD-23.02.02.iso
```

I started suspecting of a serious hardware issue, and I downloaded and installed a fresh version of Ubuntu MATE in one of the USB sticks, and it booted normally and gave me the option to install. I also went ahead and performed some RAM tests using `memtester` and I couldn't find anything problematic in terms of RAM.

Here's a video of the whole thing:

https://user-images.githubusercontent.com/1828918/217133919-f403707c-748b-4753-8664-5be35ab956db.mp4

And these are the hardware specs:

![Screenshot at 2023-02-06 22-40-20](https://user-images.githubusercontent.com/1828918/217134125-7c1e7a62-f235-4512-b43c-075275ea582b.png)

If there's anything else I can provide to help troubleshoot this issue, please let me know.

### Explain how to reproduce the bug?

Simply try to install one of the mentioned GhostBSD versions on a similar platform from an USB stick.

**Comment by jespinal** [comment]
For the record: for testing purposes, I downloaded and installed FreeBSD 13.1 DVD ISO (using one of the previously mentioned USB sticks).

I also downloaded the latest version of MemTest86 and ran all tests without issues.

**Comment by ericbsd** [comment]
Did you try another USB port?

**Comment by jespinal** [comment]
Hello, Eric:

Yes, I did, and got the same results. 

Now that you mention that, and having noticed that on FreeBSD 13.1 the wifi drivers (MediaTek chipset, wifi6) were not available, I have a new theory: probably the hardware is somewhat too new'ish (?), including all USB ports, as this model was released around 1Q from 2022.

So probably I'll need to wait for future releases of GhostBSD that are based on later versions of FreeBSD.

Btw, thanks a lot for your outstanding work :muscle: 

> Did you try another USB port?
> 
> My wife laptop does on one side of her laptop.
